apps/payments/next/tailwind.config.js

```diff
@@ -9,8 +12,321 @@ module.exports = {
     ),
     ...createGlobPatternsForDependencies(__dirname),
   ],
+  // Toggle dark-mode based on data-mode="dark"
+  darkMode: ['class', '[data-mode="dark"]'],
   theme: {
-    extend: {},
+    extend: {
```

Comment:
As L18-331 are from `fxa-react` that is currently used to be shared across packages, this should all actually go into the the shared-styles library instead of `payments-next`, correct?
cc: @StaberindeZA

Reply:
As previously noted, these styles are from `fxa-react` and will also be utilized by FXA when they migrate to next as well, so they should not be within `payments-next`. Please move this to the `shared-styles` library.